packages/vendure-plugin-customer-managed-groups/test/customer-managed-groups.spec.ts

```diff
@@ -557,6 +572,187 @@
+  });
+
+  it('Admin: make customer admin of customer managed group', async () => {
+    // adminMakeCustomerAdminOfGroupMutation
+    await adminClient.asSuperAdmin();
+
+    const { createCustomer } = await adminClient.query(addCustomer, {
+      input: {
+        firstName: 'test',
+        lastName: 'last',
+        emailAddress: `test${Math.floor(Math.random() * 1000) + 1}@hotmail.com`,
+      },
+    });
+    const { createCustomer: customer2 } = await adminClient.query(addCustomer, {
+      input: {
+        firstName: 'child',
+        lastName: 'last',
+        emailAddress: `test${Math.floor(Math.random() * 1000) + 1}@hotmail.com`,
+      },
+    });
+
+    expect(createCustomer).toMatchObject({
+      id: expect.anything(),
+    });
+
+    const { createCustomerManagedGroup: group } = await adminClient.query(
+      adminCreateCustomerManagedGroupMutation,
+      {
+        customerId: createCustomer.id,
+      }
+    );
+
+    await adminClient.query(adminAddCustomersToGroupMutation, {
+      customerGroupId: group.id,
+      customerIds: [customer2.id],
+    });
+
+    const { makeCustomerAdminOfCustomerManagedGroup: updatedGroup } =
+      await adminClient.query(adminMakeCustomerAdminOfGroupMutation, {
+        groupId: group.id,
+        customerId: customer2.id,
+      });
+
+    const updatedCustomer = updatedGroup.customers.find(
+      (c) => c.customerId === customer2.id
+    );
+    expect(updatedCustomer).toBeDefined();
+    expect(updatedCustomer.isGroupAdministrator).toBeTruthy();
+  });
+
+  it('Admin: gets orders for customer managed group', async () => {
+    await adminClient.asSuperAdmin();
+    const customersRes = await adminClient.query(customers, {
+      filter: {
+        emailAddress: {
+          eq: 'hayden.zieme12@hotmail.com',
+        },
+      },
+    });
+    const hayden = customersRes.customers.items[0];
+    expect(hayden).toBeDefined();
+    const group = hayden.groups.find((g) => g.customFields.isCustomerManaged);
+    // This is dependent on earlier unit tests
+    // const { createCustomerManagedGroup: group } = await adminClient.query(
+    //   adminCreateCustomerManagedGroupMutation,
+    //   {
+    //     customerId: hayden.id,
+    //   }
+    // );
+    expect(group).toBeDefined();
+    const { ordersForCustomerManagedGroup } = await adminClient.query(
+      adminGetOrdersForCustomerManagedGroup,
+      {
+        customerManagedGroupId: group.id,
+      }
+    );
+    // This is dependent on earlier unit tests
+    expect(ordersForCustomerManagedGroup.totalItems).toBe(2);
+    expect(ordersForCustomerManagedGroup.items[0].code).toBeDefined();
+    expect(ordersForCustomerManagedGroup.items[0].lines.length).toBeGreaterThan(
+      0
+    );
+    expect(
+      ordersForCustomerManagedGroup.items[0].payments.length
+    ).toBeGreaterThan(0);
+    expect(ordersForCustomerManagedGroup.items[0].customFields.testing).toBe(
+      'just a test'
+    );
```

Comment:
I agree. That was there from before. IMHO all unit tests should run fully independently, which is not the case now. E.g. a filtered test running just this test would fail as would the sop api order test this was copied from. One way to handle it for the part I added I supposed would be a separate describe/beforeAll function that reinits the db.